Tests/test_SeqIO_Insdc.py

```diff
@@ -67,6 +67,7 @@ def test_writing_empty_qualifiers(self):
             qualifiers={"empty": None, "zero": 0, "one": 1, "text": "blah"},
         )
         record = SeqRecord(Seq("A" * 100, generic_dna), "dummy", features=[f])
+        record.annotations["molecule_type"] = "DNA"
```

Comment:
I'd have set ``annotations = {"molecule_type": "DNA"}`` as part of the line above, but either is fine.

Reply:
Actually since you do that in some of the other tests, doing the alternative is probably good from a test coverage point of view.